src/libs/actions/Report.js

```diff
@@ -91,7 +91,7 @@ function getParticipantEmailsFromReport({sharedReportList, reportNameValuePairs,
  */
 function getSimplifiedReportObject(report) {
     const createTimestamp = lodashGet(report, 'lastActionCreated', 0);
-    const lastMessageTimestamp = moment.utc(createTimestamp).unix();
+    const lastMessageTimestamp = Date.parse(createTimestamp);
```

Comment:
this change is only used here: https://github.com/Expensify/App/blob/cc77b293287c30387d5b41b19555f962586f3201/src/libs/actions/IOU.js#L716

Which is in a function that uses the deprecated API so I think it will be refactored out anyway cc @mountiny since it's in the payment flows?

Reply:
Yes, this function should be removed I assume once the PayIOU refactors are finished.